legacy/cli/audit.go

```diff
@@ -24,6 +24,7 @@ import (
 	"github.com/sirupsen/logrus"
 
 	"sigs.k8s.io/k8s-container-image-promoter/legacy/audit"
+	"sigs.k8s.io/k8s-container-image-promoter/legacy/querycounter"
```

Comment:
I think a better name would be `requestcounter` or `reqcounter` because we'll use it to track HTTP *requests* made to GCR (in industry parlance the phrase "HTTP request" is the standard phrase, rather than "HTTP query"). Please change.

Reply:
Understood.